src/storage/query/QueryBaseProcessor.inl

```diff
@@ -171,10 +171,49 @@ cpp2::ErrorCode QueryBaseProcessor<REQ, RESP>::checkAndBuildContexts(const REQ&
 
 template<typename REQ, typename RESP>
 folly::Optional<std::pair<std::string, int64_t>>
-QueryBaseProcessor<REQ, RESP>::getTagTTLInfo(TagID tagId) {
+QueryBaseProcessor<REQ, RESP>::getTagTTLInfo(TagID tagId, bool addTagInfo) {
     folly::Optional<std::pair<std::string, int64_t>> ret;
     auto tagFound = tagTTLInfo_.find(tagId);
-    if (tagFound != tagTTLInfo_.end()) {
+
+    if (tagFound == tagTTLInfo_.end()) {
```

Comment:
You have filled the tagTTLInfo_ in buildTTLInfoAndRespSchema, right?
So the changes here is just for fetch *,  right?

I suggest you extract the common logic about filling tagTTLInfo_.   And move the function into QueryVertexPropsProcessor

Reply:
Yeah，`buildTTLInfoAndRespSchema `  function fill  the tagTTLInfo_ 
But` fetch * ` do not  do   `buildTTLInfoAndRespSchema ` in `QueryVertexPropsProcessor::process`

Reply:
`I suggest you extract the common logic about filling tagTTLInfo_. And move the function into QueryVertexPropsProcessor`
ok